setup.cfg

```diff
@@ -0,0 +1,37 @@
+[metadata]
+name = git-machete
+summary = Tool to rebase local branches with theirs parent branches
```

Comment:
I'd suggest sth sexier :) E.g. as in github repo description: `Probably the sharpest git repository organizer & rebase workflow automation tool you've ever seen ;)`

Btw I'll improve the README as well soon to reflect that paradigm shift - from rebase automation tool towards a more versatile "git repo organizer"

Reply:
OK, git, nice!